modules/i3s/docs/api-reference/i3s-loader.md

```diff
@@ -0,0 +1,114 @@
+# I3SLoader
+
+> The `I3SLoader` is experimental.
```

Comment:
Since our 3D tile loaders are maturing maybe provide a bit more color on what areas we still feel are weak?

Reply:
I'll update the doc. `i3s` has different data formats, `.pnts` and so on. Right now I only tested with MeshPyramids [I3S SPec](https://github.com/Esri/i3s-spec)